src/haz3lweb/LangDocMessages.re

```diff
@@ -1581,6 +1610,44 @@ let conapp_exp: form = {
     examples: [conapp_exp_ex],
   };
 };
+let _exp_fun = exp("e_fun");
+let _exp_arg = exp("e_arg");
+let deferred_funapp_exp_coloring_ids =
+    (~x_id: Id.t, ~arg_id: Id.t): list((Id.t, Id.t)) => [
+  (Piece.id(_exp_fun), x_id),
+  (Piece.id(_exp_arg), arg_id),
+];
+let deferred_funapp_exp: form = {
+  let explanation = {
+    message: "Partial function application. Bound part(s) of the [*function*](%i) input pattern to supplied values in the [*argument*](%i).",
```

Comment:
this message doesn't seem to be showing up correctly. just get the normal function application docs.

Reply:
I managed to let the message show up in my local build, but it doesn't work correctly on the deployed build for unknown reasons... Here comes a screenshot of the message:
![image](https://github.com/hazelgrove/hazel/assets/108375845/f348ef06-3c16-4aec-bff0-a86bcd8d23a1)